src/index.html

```diff
@@ -19,9 +19,382 @@
+      rel="stylesheet"
+    />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__line">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__line">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__line">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__line">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__line">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__line">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__line">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__line">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Imac"
+          class="card__image"
+        />
+
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__text">Product code: 195434</p>
+
+        <div class="rating">
+          <div class="rating__block">
+            <ul class="rating__list">
+              <li class="rating__stars"></li>
+              <li class="rating__stars"></li>
+              <li class="rating__stars"></li>
+              <li class="rating__stars"></li>
+              <li class="rating__stars"></li>
+            </ul>
+            <p class="rating__text">Reviews: 5</p>
+          </div>
+        </div>
+
+        <div class="price">
+          <div class="price__block">
+            <p class="price__text">Price:</p>
+            <p class="price__count">$2,199</p>
+          </div>
+        </div>
+        <button
+          class="card__button"
```

Comment:
The `data-qa` attribute is used for test selectors. Make sure it's necessary here, otherwise, it could be removed to clean up the HTML if not used for testing purposes.